packages/draw/example/main.ts

```diff
@@ -15,23 +16,32 @@ document.addEventListener('DOMContentLoaded', async () => {
     );
 
     const artist = new Artist({
-      data,
       style: {
         nodeColor: '#01b0d3',
         linkColor: '#01586a',
         titleColor: '#ffffff',
       },
+      canvas,
+    });
+
+    const simulation = new Simulation({
+      data,
       simulationConfig: {
         randomizeStartingPoints: false,
       },
-      canvas,
+      width: artist.canvasInitialWidth,
+      height: artist.canvasInitialHeight,
     });
 
+    simulation.start([
+      (nodes, links) => {
+        artist.draw(nodes, links);
+      },
+    ]);
+
     artist.addEventListener('nodeClick', (node) => {
       alert(node.id);
     });
-
-    artist.draw();
   } catch (error) {
```

Comment:
I'm wondering if it would be ideal to wrap this in a single function or something for users of the library? I like the abstractions, but this seems like a lot of code for the average person to get up and running, and every time we change these internal-ish classes it will be a major version bump.

Reply:
I think this is resolved by https://github.com/jollyjerr/mindgraph/pull/12/commits/d98e3c2cd7df743f57014d2c3f81162b9a56ab9c. It's a proposal of a top-level user friendly class called `MindGraph` that could represent the common user scenario for people who don't want to muck around with any of the internal classes - Simulation, Artist, Canvas, RenderableNode, etc